chaos-proxy/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "dydx-cli",
```

Comment:
Just out of curiosity what does `dydx` stand for?

Reply:
It wasn't changed from the original code I've copied over (the link to the original is in the readme)